processes/releasing.md

```diff
@@ -0,0 +1,85 @@
+# Releasing
+
+WIP: This document is work in progress.
+
+## Process
+
+High-level release process we followed in case it makes it clearer:
+
+1. Uplift IPAM to use latest CAPI
+1. [Release IPAM](https://github.com/metal3-io/ip-address-manager/blob/main/docs/releasing.md)
+1. Uplift BMO to use latest go modules as in CAPI (controller runtime,
+   `k8s.io/*` in `go.mod`)
+1. [Release BMO](https://github.com/metal3-io/baremetal-operator/blob/main/docs/releasing.md)
+1. Uplift CAPM3 to use latest CAPI, IPAM, BMO (in `go.mod`)
+1. [Release CAPM3](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/releasing.md)
+1. Update Metal3 dev-env and CI to support new releases
+
+## Update Metal3 dev-env and CI to support new releases
+
+If a new minor or major was released, we need to make changes in Metal3 dev-env
+and CI.
+
+### Tag Ironic and Mariadb images
+
+After releasing v1.x.y version of CAPM3, create an annotated tag with `capm3-`
+prefix + release version in
+[Ironic-image](https://github.com/metal3-io/ironic-image) and
+[mariadb-image](https://github.com/metal3-io/mariadb-image) Github
+repositories.
+
+```bash
+git tag -s -a capm3-v1.x.y -m capm3-v1.x.y [optional sha, or tag^{}]
+git push origin capm3-v1.x.y
+```
+
+After this, [Ironic](https://quay.io/repository/metal3-io/ironic) and
+[Mariadb](https://quay.io/repository/metal3-io/mariadb) container images will
+be built automatically  with `capm3-v1.x.y` tag in Quay. Verify the build is
+triggered, as often Quay has disabled the build trigger due build failures or
+other reasons.
+
+### Create Jenkins jobs
+
+Two Jenkins jobs need to be created in
+[JJB](https://gerrit.nordix.org/plugins/gitiles/infra/cicd/+/refs/heads/master/jjb/metal3/):
+
+- a periodic job that runs on a regular basis.
+- a PR verification job that is triggered by a keyword on a PR targeted for that
+  release branch.
+
+We also need to change project-infra keywords for required tests for some of
+the repositories.
+
+### Dev-env changes
+
+Metal3-dev-env variables need to be modified. After a major or minor release,
```

Comment:
Maybe provide an example or at least the line(s) to change?

Reply:
That's the plan. This PR is marked WIP as I'm documenting the required changes as were doing them in 1.4 release cycle.

Reply:
Done.